python/graphstorm/inference/lp_infer.py

```diff
@@ -42,6 +42,7 @@ class GSgnnLinkPredictionInferrer(GSInferrer):
     def infer(self, data, loader, save_embed_path,
             edge_mask_for_gnn_embeddings='train_mask',
             use_mini_batch_infer=False,
+            infer_batch_size=1024,
```

Comment:
You need to add this new argument at the end of the argument list to make it backward compatible.

Reply:
Also you need to add a docstring